pwndbg/checksec.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+ Wrapper to checksec utility.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import sys
+import subprocess
+
+import pwndbg.file
+
+def checksec():
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+    result = {}
+
+    for program in ['checksec', 'checksec.sh']:
+        program = pwndbg.which.which(program)
+        if program:
+            cs_out = subprocess.check_output([program, '--file', local_path]).decode('utf-8')
+
+            if "Full RELRO" in cs_out:
+                result['RELRO'] = 2
```

Comment:
We should probably use something like enum here (e.g. `from enum import Enum`).

This would however require us to add `enum34` into `requirements.txt` (as enum is a package for Python 2). I am not sure if we want to do that. @zachriggle is it okay?

Then it might look something like:
```python
class Relro(Enum):
    NO = 0
    PARTIAL = 1
    FULL = 2
    
    @staticmethod
    def from_checksec_str(string):
        # ...
```